stereo_image_proc/src/stereo_image_proc/point_cloud_node.cpp

```diff
@@ -101,6 +101,7 @@ PointCloudNode::PointCloudNode(const rclcpp::NodeOptions & options)
   int queue_size = this->declare_parameter("queue_size", 5);
   bool approx = this->declare_parameter("approximate_sync", false);
   this->declare_parameter("use_system_default_qos", false);
+  this->declare_parameter("avoid_point_cloud_padding", false);
```

Comment:
It would be nice to document why you might not want to avoid padding. Maybe with a `ParameterDescriptor` argument?

I'm not sure if we any solid documentation for ROS 2 usage of `image_pipeline`, yet.

Reply:
Done